packages/dm-core-plugins/src/view_selector/ViewSelectorPlugin.tsx

```diff
@@ -147,6 +147,12 @@ export const ViewSelectorPlugin = (
         />
       )}
       <Content
+        style={{
+          ...(internalConfig.asSidebar
+            ? { paddingLeft: '8px' }
+            : { paddingTop: '8px' }),
+          paddingRight: '8px',
+        }}
```

Comment:
Why pass this? Couldn't you achieve the same by making a styled-component with a prop out of Content? Or wrapping Content in a div with style?

Reply:
Force of habit. Not used to the concept of styled components approach.

> Or wrapping Content in a div with style?

I'm just more used to this approach (although not having to take in styles as a prop...)